docs/labs/lab03-configlet-management/config_search.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2022 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the COPYING file.
+
+from cvprac.cvp_client import CvpClient
+import ssl
+ssl._create_default_https_context = ssl._create_unverified_context
+import requests.packages.urllib3
+requests.packages.urllib3.disable_warnings()
+
+# Create connection to CloudVision
+clnt = CvpClient()
+clnt.connect(nodes=['cvp1'], username="username",password="password")
+
+def main():
+
+   print('Retrieving configlets ...')
+
+   inventory = clnt.api.get_inventory()
+   configlets = clnt.api.get_configlets_and_mappers()['data']['configlets']
+   generated_configlet_mappers = clnt.api.get_configlets_and_mappers()['data']['generatedConfigletMappers']
+   configlet_mappers = clnt.api.get_configlets_and_mappers()['data']['configletMappers']
+
+   print('Number of configlets:', len(configlets))
+
+   searchAgain = True
+   while searchAgain:
+        try:
+            search = input( "\nEnter Config Line: " )
+            print(f"\n\n\'{search}\' has been found in following configlets:\n\n")
+            print(f"{'Hostname':<30}{'Serial number':<50}{'MAC address':<30}{'Configlets':<40}")
+            print("=" * 150)
+            for i in inventory:
+                device = i['hostname']
+                device_sn = i['serialNumber']
+                device_mac = i['systemMacAddress']
+                configlet_list = []
+                for c in configlets:
+                    for g in generated_configlet_mappers:
+                        if device_mac == g['netElementId'] and c['key'] == g['configletBuilderId'] and search in c['config']:
+                            configlet_list.append(c['name'])
+                    for k in configlet_mappers:
+                        if device_mac == k['objectId'] and c['key'] == k['configletId'] and search in c['config']:
+                            configlet_list.append(c['name'])
```

Comment:
Can you explain the comparison between the configlet and configletBuilderId from generated_configlet_mappers and the comparision to configletId for configlet_mappers?

Reply:
Can you please address my questions in the previous comment?

Reply:
Discussed unicast.